www/docs/user/confirm/index.php

```diff
@@ -27,12 +27,14 @@
         if ($THEUSER->isloggedin()) {
             ?>
 
-    <p>Hi, and welcome to TheyWorkForYou! You are now logged in.</p>
+    <p>Welcome to TheyWorkForYou! You are now logged in.</p>
 
-    <p><strong>TheyWorkForYou</strong> helps make sense of this vital democratic resource.
-    Feel free to use it to keep an eye on <strong>your MP</strong>, Peers, or other representatives covered.
+    <p><strong>TheyWorkForYou</strong> helps make sense of the goings on in Parliament: debates, divisions, committees and questions.</p>
 
-    <p>If you are looking to sign up for our API, please <a href="/api/key">get one here</a>.</p>
+    <p>You can use it to keep an eye on your MP, Peers, or any of the other representatives we cover: see what speeches they make, how they vote, and check how well you're being represented.
+        And if you have opinions about any of that? You can look for the &lsquo;Send a message&rsquo; button at the top of your MP's page.</p>
```

Comment:
There are still two straight quotes on this line and the line above, I'm afraid.